### PR TITLE
Implement DeskBand functionally.

### PR DESCRIPTION
**Pull request by joeltorres-7**
# Description

This branch implements the Deskband functionality that was requested on the issue #429. In order to accomplish this, I added 3 new icons on the `/public/img/icon/ui` path, converted the initial icon element to a div, and implemented a new data-action (BANDTOGG | BANDHIDE) for the element.

# Preview

win11-deskband

## Type of change

Please delete options that are not relevant.

- [ x ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by joeltorres-7** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by yashash1511** [comment]
image
clicking on spotify doesn't open it

can you implement functionality that when we click on the apps in the deskband it opens them

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "search.useIgnoreFiles": true
```
Comment (yashash1511) [inline .vscode/settings.json:2]: don't add this file

**Comment by yashash1511** [comment]
image

please don't add this it is already present in [public/img/icon/spotify.png](https://github.com/blueedgetechno/win11React/blob/master/public/img/icon/spotify.png)

File: src/components/start/index.js [hunk 2]
```diff
@@ -70,6 +70,24 @@ export const DesktopApp = () => {
...
+  const sidepane = useSelector((state) => state.sidepane);
+  
+  return (
+    <div
+      className="bandpane dpShad"
+      data-hide={sidepane.banhide}
+      style={{ "--prefix": "BAND" }}
+    >
+      <div className="bandContainer">
+        <Icon className="hvlight" src="defender" ui width={17} />
```
Comment (yashash1511) [inline src/components/start/index.js:83]: once you move the icon to public/img/icon you will have to remove the ui prop

**Comment by yashash1511** [comment]
[Allow edits by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

it would be nic if you allowed this (present in the right sidebar) so we can make few small changes required sometimes.

File: src/components/start/index.js (hunk 2 above)
Comment (yashash1511) [reply src/components/start/index.js]: fixed

**Comment by yashash1511** [comment]
Nice work Joel

**Pull request merged**